railties/lib/rails/commands/console/console_command.rb

```diff
@@ -73,6 +73,10 @@ class ConsoleCommand < Base
       class_option :environment, aliases: "-e", type: :string,
         desc: "Specifies the environment to run this console under (test/development/production)."
 
+      def help
+        self.class.command_help(shell, self.class.command_name)
+      end
```

Comment:
Seems like app commands (when we get to adding those could run in to this problem too), can we move this to the `Rails::Command::Base` class and just make it the default?